website/frontend/templates/changelog.html

```diff
@@ -27,5 +27,5 @@
         {% endif %}
       {% endfor %}
       </div>
-    </div><!-- /.container -->
```

Comment:
Why is this being moved to the end of the file? You've done this in all the files.